crates/bevy_ecs/src/system/exclusive_function_system.rs

```diff
@@ -65,11 +65,6 @@ where
         self.system_meta.name.clone()
     }
 
-    #[inline]
-    fn type_id(&self) -> TypeId {
```

Comment:
The removal of these methods is breaking. Still good to do, but should be in a migration guide.

Reply:
These methods are still available with the same API (they're provided by `System`), they just produce a different value. I'll still add an explicit migration guide.